.travis.yml

```diff
@@ -31,6 +39,22 @@ matrix:
   allow_failures:
     - os: osx
 script: |
+  set -e
```

Comment:
The set -e, I presume, is so that you explode in the middle of the git statements at the first failure rather than trying to run subsequent ones?